client/src/pages/DonorHomepage.tsx

```diff
@@ -2,17 +2,17 @@ import React, { FunctionComponent } from "react";
 import Container from "react-bootstrap/Container";
 import Row from "react-bootstrap/Row";
 
+import BannerCarousel from "../components/atoms/BannerCarousel";
 import DonorImpactSection from "../components/organisms/DonorImpactSection";
 import DonorPage from "../components/layouts/DonorPage";
 import DonorRequestGroupBrowser from "../components/organisms/DonorRequestGroupBrowser";
-import HomepageBanner from "../components/organisms/HomepageBanner";
 
 const DonorHomepage: FunctionComponent = () => {
     return (
         <Container className="donor-homepage" fluid>
             <DonorPage>
                 <Row>
-                    <HomepageBanner />
+                    <BannerCarousel />
```

Comment:
Make a wrapper for `BannerCarousel` that reads `DonorHomepageConfig` and use it here instead of `BannerCarousel`